stdlib/public/Concurrency/Task.cpp

```diff
@@ -21,20 +21,129 @@
 #include "TaskPrivate.h"
 
 using namespace swift;
+using FutureFragment = AsyncTask::FutureFragment;
+
+size_t FutureFragment::fragmentSize(const Metadata *resultType) {
+  return storageOffset(resultType) + resultType->vw_size();
+}
```

Comment:
I'm not sure why this is out-of-line give that `storageOffset` isn't.

Reply:
No reason, fixed!